internal/store/serverless_instances.go

```diff
@@ -32,10 +32,14 @@ type ServerlessInstanceDescriber interface {
 	ServerlessInstance(string, string) (*atlas.Cluster, error)
 }
 
+type ServerlessInstanceDeleter interface {
+	DeleteServerlessInstance(string, string) error
+}
+
 // ServerlessInstances encapsulates the logic to manage different cloud providers.
 func (s *Store) ServerlessInstances(projectID string, listOps *atlas.ListOptions) (*atlas.ClustersResponse, error) {
 	switch s.service {
-	case config.CloudService:
+	case config.CloudService, config.CloudGovService:
```

Comment:
I think that serverless is currently not available on fedramp. I have checked on `https://cloud-dev.mongodbgov.com/` and I don't have it on the UI